### DISCUSSION TITLE
Toolbar does not come with the icon

### DISCUSSION THREAD

**Discussion by luapfr** [rustdesk/rustdesk#7715] [General]
### Bug Description

See the attached image to understand better

![RustDesk Error](https://github.com/rustdesk/rustdesk/assets/50739062/b3f57c55-0e1a-4c32-b997-f9a549369fb7)

### How to Reproduce

See the attached image to understand better

### Expected Behavior

See the attached image to understand better

### Operating system(s) on local side and remote side

Windows 7, 10, 11 etc...

### RustDesk Version(s) on local side and remote side

All

### Screenshots

![RustDesk Error](https://github.com/rustdesk/rustdesk/assets/50739062/8cd71d62-ddda-4732-9cb7-f24c4114c834)

**Comment by garbb** [comment]
Just wanted to mention that this bug still appears to be present in the latest version 1.4.3 even though the associated issue https://github.com/rustdesk/rustdesk/issues/7713 has been closed.
I understand if this is a low priority issue since it doesn't really effect the functionality of anything.
